apps/deploy-web/env/.env.sample

```
@@ -45,3 +45,9 @@ NEXT_PUBLIC_UNLEASH_FRONTEND_API_TOKEN=
 
 UNLEASH_SERVER_API_URL=
 UNLEASH_SERVER_API_TOKEN=
+
+NEXT_PUBLIC_TURNSTILE_ENABLED=true
+# https://developers.cloudflare.com/turnstile/troubleshooting/testing/#dummy-sitekeys-and-secret-keys
+NEXT_PUBLIC_TURNSTILE_SITE_KEY=1x00000000000000000000AA
+TURNSTILE_SECRET_KEY=1x0000000000000000000000000000000AA
+E2E_TESTING_CLIENT_TOKEN=any-random-string
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The E2E_TESTING_CLIENT_TOKEN key should go before the NEXT_PUBLIC_TURNSTILE_ENABLED key

(UnorderedKey)